clang/test/SemaSYCL/args-size-overflow.cpp

```diff
@@ -13,10 +13,10 @@ __attribute__((sycl_kernel)) void kernel(F KernelFunc) {
 template <typename Name, typename F>
 void parallel_for(F KernelFunc) {
 #ifdef GPU
-  // expected-warning@+8 {{kernel argument count (2001) exceeds supported maximum of 2000 on GPU}}
+  // expected-warning@+8 {{kernel arguments size (7994 bytes) exceeds supported maximum of 2048 on GPU}}
   // expected-note@+7 {{array elements and fields of a class/struct may be counted separately}}
```

Comment:
Also, the "note" message no longer makes sense as it does not make a difference due to decomposition.

Reply:
Done.